llvm/lib/Object/Minidump.cpp

```diff
@@ -142,6 +164,15 @@ MinidumpFile::create(MemoryBufferRef Source) {
       continue;
     }
 
+    // We treat exceptions differently here because the LLDB minidump
+    // makes some assumptions about uniqueness, all the streams other than
```

Comment:
I'm not sure which assumptions are you referring to (from what I can tell it's this code here, which is assuming there can be only one stream of a given type), but it'd be nice if this could be explained in a way which does not involve LLDB (say what do we want to do, not who we're doing it for)